### ISSUE TITLE
FAIL in matrixMul example

### ISSUE THREAD

**Issue by vvolkl** [alpaka-group/cupla#212]
The matrixMul example fails for several CPU backends, for example `-DALPAKA_ACC_CPU_B_OMP2_T_SEQ_ENABLE=ON`:

```
...
Error! Matrix[51197]=0.00000000, ref=1.59999990 error term is > 1.000000E-06
Error! Matrix[51198]=0.00000000, ref=1.59999990 error term is > 1.000000E-06
Error! Matrix[51199]=0.00000000, ref=1.59999990 error term is > 1.000000E-06
Result = FAIL

Note: For peak performance, please refer to the matrixMulCUBLAS example.

```

Looking at the CI setup, it seems like this is expected, is that correct? If that is the case, can I update the [install instructions](https://github.com/alpaka-group/cupla/blob/master/INSTALL.md#compile-an-example) to use the vectorAdd example instead?

**Comment by sbastrakov** [comment]
Good catch, thanks for reporting. I am a bit confused that [this comment](https://github.com/alpaka-group/cupla/blob/16fbd81a1023720ccfd50135354e0d7175fbbfbc/script/run_test.sh#L64-L66) implies that `ALPAKA_ACC_CPU_B_OMP2_T_SEQ` should work, but the following condition is different. I will look into that

**Comment by sbastrakov** [comment]
Ah, but the documentation you linked even contradicts this comment. So some things among those are wrong for sure

**Comment by psychocoderHPC** [comment]
Unfortionally I never added the element layer to the example. The required changes are in the GTC video linked in the README.md.
We should update the example.

**Comment by sbastrakov** [comment]
So does changing `CUPLA_KERNEL` to `CUPLA_KERNEL_OPTI` (as suggested by the docs) not help here?

**Comment by psychocoderHPC** [comment]
No it is not enough because the kernel does not use the element layer. The element layer must be added then you can use OPTI kernel call.

Am 20. August 2021 11:59:42 MESZ schrieb Sergei Bastrakov ***@***.***>:
